tests/test_definitions/test_expectations_v3_api.py

```diff
@@ -59,408 +59,414 @@ def pytest_generate_tests(metafunc):  # noqa C901 - 35
         )
         for backend in backends:
             for filename in test_configuration_files:
-                pk_column: bool = False
```

Comment:
@TrangPham If these improvements (the use of `with`) are not in support of solving the decimals issues, then it would probably be easier to handle in a separate pull request -- or does it belong as part of this one?  Thank you.

Reply:
I can move it to another pull request. This was just something small I found an improvement for.